tools/image-builder/build_bond.zsh

```diff
@@ -0,0 +1,82 @@
+#!/bin/zsh
+
+set -eux
+setopt shwordsplit
+
+HOME=$1
```

Comment:
This is shadowing a common envvar. `SYMLINKED_HOME`?

Reply:
Fixed.